ui/app/components/job-page/parts/title.js

```diff
@@ -38,7 +39,8 @@ export default Component.extend({
       job.set('status', 'running');
     } catch (err) {
       let message = messageFromAdapterError(err);
-      if (!message || message === 'Forbidden') {
+
+      if (err instanceof ForbiddenError) {
```

Comment:
This is unrelated to your PR but it made me wonder why we can’t know in advance that a token doesn’t have this permission. I didn’t find anything that would clearly relate to that in the list of capabilities 🤔

```
  "list-jobs",
  "read-job",
  "csi-list-volume",
  "csi-read-volume",
  "read-job-scaling",
  "list-scaling-policies",
  "read-scaling-policy",
  "scale-job",
  "submit-job",
  "dispatch-job",
  "read-logs",
  "read-fs",
  "alloc-exec",
  "alloc-lifecycle",
  "csi-mount-volume",
  "csi-write-volume"
```

Reply:
Looks like both [Stop Job](https://www.nomadproject.io/api-docs/jobs/#stop-a-job) and [Promote Deployment](https://www.nomadproject.io/api-docs/deployments/#promote-deployment) use the `submit-job` acl.

Reply:
ahhhhh yes, thanks for the links 😯